### ISSUE TITLE
Event drag & drop support

### ISSUE THREAD

**Issue by TatsuUkraine** [JonasWanke/timetable#21]
Great package!

I didn't find this feature been listed, but it's good to have drag&drop feature to the event items with the ability to define drag steps. So the event could be dragged on the time that multiples of 15 min like: 10:00, 10:15, 10:30 etc.

Or maybe provide a callback to event instance, that receives current hover date time and expects bool to be returned, and if it's FALSE - event can't be dragged over this time slot.

**Comment by JonasWanke** [comment]
You're right, drag & drop currently isn't supported but is an important feature I'd like this package to support. I also like the idea of a customizable step size.

Unfortunately, my calendar is pretty full at the moment, so I guess I'll only be able to start implementing this feature in about two weeks.

**Comment by TatsuUkraine** [comment]
thanks for an update)

**Comment by mbartnik** [comment]
Hello, any updates on this feature? I tried implementing it myself but not in the library, just trying to wrap my Event in Draggable widget. I am stumped on step dragging i.e. moving event only in 15 min increases. I can't see any way to control the dragging. If anyone has any ideas, I would be willing to implement this and make a PR.

**Comment by JonasWanke** [comment]
I'm planning to have a look into this as part of the major refactor in #17.
